schemas/repositories.json

```diff
@@ -9,6 +9,17 @@
         "type": "string",
         "pattern": "^[a-z][a-z-]{0,1023}$"
       },
+      "description": {
+        "type": "string"
+      },
+      "channel": {
+        "type": "string",
+        "enum": ["release", "beta", "nightly", "esr"]
```

Comment:
Opting to be very strict about this right now (corresponding to: https://docs.telemetry.mozilla.org/concepts/channels/channel_normalization.html) since it covers the existing set of possible channels. We can always update this in the future if required, but this will hopefully act as a nudge to use consistent terminology across Mozilla, which I think is desirable.